### PR TITLE
Refs 101: add PR title validation

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMSCONTENT-101

**Comment by jlsherrill** [comment]
@Andrewgdewar you okay with this? https://github.com/content-services/content-sources-frontend/pull/12

**Comment by jlsherrill** [comment]
⚠️ This task currently requires qe-approval, but this PR does not fully resolve the task.  Please contact QE to determine appropriate testing required.

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/14709226

File: .github/workflows/title.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    types: [opened, edited, synchronize, reopened]
+
+jobs:
+  title-validate:
+    name: Title
+    runs-on: ubuntu-latest
+    steps:
+    - uses: deepakputhraya/action-pr-title@master
+      with:
+        regex: '(Fixes |Refs )\d+: .+' # Regex the title should match.
```
Comment (Andrewgdewar) [inline .github/workflows/title.yml:13]: Right now we are using CONTENT-41 for example in the commit messages/PR's in the front-end.  This proposes that we change this to Katello's syntax correct?

**Review by Andrewgdewar — APPROVED** [review]
LGTM

File: .github/workflows/title.yml (hunk 1 above)
Comment (jlsherrill) [reply .github/workflows/title.yml:13]: Yep, this was sorta decided the week that you were out, the reason is that we wanted to associate more than one pr with an issue, and be able to indicate whether the pr actually 'fixes' the issue, or is a secondary change

**Pull request merged**